coderd/coderd.go

```diff
@@ -1502,6 +1502,11 @@ func (api *API) CreateInMemoryTaggedProvisionerDaemon(dialCtx context.Context, n
 		dbTypes = append(dbTypes, database.ProvisionerType(tp))
 	}
 
+	keyID, err := uuid.Parse(string(codersdk.ProvisionerKeyIDBuiltIn))
+	if err != nil {
+		return nil, xerrors.Errorf("failed to parse built-in provisioner key ID: %w", err)
+	}
```

Comment:
```suggestion
	keyID := uuid.MustParse(codersdk.ProvisionerKeyIDBuiltIn)
```

1. a built-in constant we control feels like a good target for `uuid.MustParse`
2. these are already just basic strings, yeah?

```go
const (
	ProvisionerKeyIDBuiltIn  = "00000000-0000-0000-0000-000000000001"
	ProvisionerKeyIDUserAuth = "00000000-0000-0000-0000-000000000002"
	ProvisionerKeyIDPSK      = "00000000-0000-0000-0000-000000000003"
)
```

Reply:
They should be safe, but I'd rather just follow the no panic principal here so we at least return a proper error in the response